packages/components/ovh-reket/src/index.js

```diff
@@ -2,7 +2,13 @@ import { Reket } from '@ovhcloud/reket-core';
 import { AxiosReketClient } from '@ovhcloud/reket-axios-client';
 
 import { DEFAULT_REQUEST_TYPES } from './constants';
-import { redirectToLoginPage, redirectToLogoutPage } from './redirections';
+import {
+  redirectToLoginPage as toLogin,
+  redirectToLogoutPage as toLogout,
+} from './redirections';
+
+export const redirectToLoginPage = toLogin;
+export const redirectToLogoutPage = toLogout;
```

Comment:
`import { redirectToLoginPage, redirectToLogoutPage } from './redirection`
`export redirectToLoginPage`
`export redirectToLogoutPage`

Doesn't work? you are forced to work with aliases?

Reply:
```js
export redirectToLoginPage; // Unexpected token (missing const)
```
```js
export const redirectToLoginPage; // Identifier 'redirectToLoginPage' has already been declared
```
it's kind of ugly but i'm not sure if there are other ways around that

Reply:
Something like:
```
export * from './redirections';
``` ?